pkg/conference/participant/tracker.go

```diff
@@ -161,7 +161,11 @@ func (t *Tracker) RemovePublishedTrack(id track.TrackID) {
 }
 
 // Subscribes a given participant to the track.
-func (t *Tracker) Subscribe(participantID ID, trackID track.TrackID, requirements track.TrackMetadata) error {
+func (t *Tracker) Subscribe(
+	participantID ID,
+	trackID track.TrackID,
+	desiredWidth, desiredHeight int,
+) error {
```

Comment:
This change doesn't seem related?

Reply:
I changed the argument from `TrackMetadata` to `desiredWidth, desiredHeight`. Previously these were the only fields in the `TrackMetadata`. Now, it's more extensive (contains the mute state and probably will contain more), i.e. it was required since I changed the input parameters for the `published.Subscribe()` function.